src/articles/ArticlePreview.js

```diff
@@ -127,14 +130,23 @@ export default function ArticlePreview({
           />
         </sweetM.TagsOfInterests>
       )}
-
       <SmallSaveArticleButton
         article={article}
         isArticleSaved={isArticleSaved}
         setIsArticleSaved={setIsArticleSaved}
       />
+      <s.TitleContainer>
+        <s.Title className={article.opened ? "opened" : ""}>
+          {titleLink(article)}{" "}
+        </s.Title>
+        <ReadingCompletionProgress
+          value={readingCompletion}
+          style={
```

Comment:
same story as above: move inside of ReadingCompletionProgress? Could be argued that it should know how to render itself.